packages/fxa-content-server/server/lib/glean/index.js

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const serverGleanEvents = require('./server_events');
+
+let appConfig;
+let gleanEventLogger;
+let gleanServerEventLogger;
+
+module.exports = function (config) {
+  appConfig = config;
+
+  gleanEventLogger = serverGleanEvents.createAccountsEventsEvent({
+    applicationId: config.serverGleanMetrics.applicationId,
+    appDisplayVersion: config.version,
+    channel: config.serverGleanMetrics.channel,
+    logger_options: { app: config.serverGleanMetrics.loggerAppName },
+  });
+  gleanServerEventLogger = serverGleanEvents.createEventsServerEventLogger({
+    applicationId: config.serverGleanMetrics.applicationId,
+    appDisplayVersion: config.version,
+    channel: config.serverGleanMetrics.channel,
+    logger_options: { app: config.serverGleanMetrics.loggerAppName },
+  });
+
+  return {
+    rp: {
+      formView: async (metrics, rpFormType) => {
```

Comment:
I added an event so I can verify that something is showing up in the log.  The impl here and the call site will be updated in a follow-up patch.